src/beancount.pest

```diff
@@ -38,9 +38,9 @@ valid_non_letter_commodity_char = { QUOTATION_MARK |  "_" | HYPHEN | "." }
 commodity = @{ ASCII_ALPHA_UPPER ~ (ASCII_ALPHA_UPPER | valid_non_letter_commodity_char){0, 22} ~ (ASCII_ALPHA_UPPER | ASCII_DIGIT)? }
 
 //// Account primitives
-account_type = { "Assets" | "Liabilities" | "Income" | "Expenses" | "Equity" }
-account_name_piece = { ":" ~ (ASCII_ALPHA_UPPER | ASCII_DIGIT) ~ (ASCII_ALPHA_LOWER | ASCII_ALPHA_UPPER | ASCII_DIGIT | "-")* }
-account = @{ account_type ~ account_name_piece* }
+account_type = { UPPERCASE_LETTER ~ (LETTER | DECIMAL_NUMBER | "-")* }
```

Comment:
Huh, I didn't know the account type could be user-defined.  I was working off that assumption based on [this documentation](https://docs.google.com/document/d/1wAMVrKIA2qtRGmoVDSUBJGmYZSygUaR0uOMW1GV3YE0/edit#heading=h.17ry42rqbuiu), but seems like these docs are out-of-date.  Looking at the actual definition of the account confirms you are right - https://bitbucket.org/blais/beancount/src/487cb9b0248427ac6882889b6b8bcc1665ce393e/beancount/core/account.py?at=default&fileviewer=file-view-default#account.py-27.  Good catch!

Reply:
It's documented in the options reference: https://docs.google.com/document/d/1_-T_BvDtUjj9M7liZMNSkrL8pgC60TGMBlYCiV1e4ZM/